app/src/main/assets/map_theme/jawg/streetcomplete.yaml

```diff
@@ -146,20 +156,29 @@ layers:
     streetcomplete_track2:
         data: { source: streetcomplete_track2 }
         current:
-            filter: { old: [false] }
-            draw:
-                track-lines:
-                    color: global.track_color
-                    width: [[14, 6px],[18, 12px]]
-                    collide: false
-                    join: round
-                    order: 1000
+          filter: { old: [ false ], record: [ false ] }
```

Comment:
Can you restore the previous indentation (4 spaces) for all your changes in this file please?

Reply:
Not sure why it auto formats to 2spaces. I have made it all 4space hopefully.